Error while checking build/O2/fullCI for e235c7e9aefc0ca50f7473bc2c209f7ad78bb87f at 2023-08-11 11:31:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/QualityControl-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'


## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
/sw/SOURCES/O2/11750-slc8_x86-64/0/Detectors/ITSMFT/MFT/base/src/Ladder.cxx:84:37: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/11750-slc8_x86-64/0/Detectors/ITSMFT/MFT/base/src/Ladder.cxx:149:37: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/11750-slc8_x86-64/0/Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx:89:35: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/11750-slc8_x86-64/0/Detectors/ITSMFT/MFT/simulation/src/Detector.cxx:286:37: error: statement should be inside braces [readability-braces-around-statements]
/sw/SOURCES/O2/11750-slc8_x86-64/0/Detectors/ITSMFT/MFT/simulation/src/Detector.cxx:293:37: error: statement should be inside braces [readability-braces-around-statements]
++ [[ 0 == 0 ]]
++ exit 1
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11750/e235c7e9aefc0ca50f7473bc2c209f7ad78bb87f/build_O2_fullCI/pretty.html).

--- a/Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx
+++ b/Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx
@@ -86,9 +86,9 @@ HeatExchanger::HeatExchanger()
 
   // for the chips aligment --> decrease of the HE thickness then increase of the rohacell density
   Double_t decreaseHE = 0.0;
-  if (mftBaseParam.buildAlignment)
+  if (mftBaseParam.buildAlignment) {
     decreaseHE = 0.4;
-
+  }
   mHeatExchangerThickness = 1.398 - decreaseHE;
   mCarbonThickness = (0.0290) / 2.; // half thickness of the carbon plate
   initParameters();

--- a/Detectors/ITSMFT/MFT/base/src/Ladder.cxx
+++ b/Detectors/ITSMFT/MFT/base/src/Ladder.cxx
@@ -81,9 +81,9 @@ TGeoVolume* Ladder::createVolume()
 
     // to perform the alignment
     Double_t shiftFlex = 0.0;
-    if (mftBaseParam.buildAlignment)
+    if (mftBaseParam.buildAlignment) {
       shiftFlex = 0.2;
-
+    }
     TGeoVolumeAssembly* flexVol = mFlex->makeFlex(mSegmentation->getNSensors(), flexLength);
     mLadderVolume->AddNode(flexVol, 1, new TGeoTranslation(flexLength / 2 + Geometry::sSensorSideOffset / 2, shiftY, Geometry::sFlexThickness / 2 - 2 * (Geometry::sKaptonOnCarbonThickness + Geometry::sKaptonGlueThickness) - shiftFlex));
   }
@@ -146,9 +146,9 @@ void Ladder::createSensors()
 
     // To perform chips alignment --> shift of the chip glue layer
     Double_t shiftGlue = 0.0;
-    if (mftBaseParam.buildAlignment)
+    if (mftBaseParam.buildAlignment) {
       shiftGlue = 0.19;
-
+    }
     Double_t posglue[3] = {SegmentationAlpide::SensorSizeCols / 2., SegmentationAlpide::SensorSizeRows / 2.,
                            Geometry::sGlueThickness / 2 - Geometry::sChipThickness + shiftGlue - 2 * (Geometry::sKaptonOnCarbonThickness + Geometry::sKaptonGlueThickness)}; // fm
 

--- a/Detectors/ITSMFT/MFT/simulation/src/Detector.cxx
+++ b/Detectors/ITSMFT/MFT/simulation/src/Detector.cxx
@@ -283,15 +283,17 @@ void Detector::createMaterials()
     //  smaller rohacell thickness, greater density by 15% (from 1.327cm to 1.117cm)
     dRohacell = 0.032 / (1 - 0.158);
     // to perform chips aligment
-    if (mftBaseParam.buildAlignment)
+    if (mftBaseParam.buildAlignment) {
       dRohacell = dRohacell / (1 - 0.358); // decrase by 4mm of the rohacell thickness
+    }
   }
   if (Geometry::sGrooves == 1) {
     // With grooves, usual rohacell density: 0.032 g/cm3 rohacell 31
     dRohacell = 0.032;
     // to perform chips aligment
-    if (mftBaseParam.buildAlignment)
+    if (mftBaseParam.buildAlignment) {
       dRohacell = dRohacell / (1 - 0.301); // decrase by 4mm of the rohacell thickness
+    }
   }
 
   // Polyimide pipe mixture

